charts/base-cluster/values.schema.json

```diff
@@ -1263,6 +1263,81 @@
       },
       "additionalProperties": false
     },
+    "weaveworks": {
+      "type": "object",
+      "description": "Values for the gitops-chart",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "adminUser": {
+          "type": "object",
+          "properties": {
+            "create": {
```

Comment:
I would drop this `create` flag, the user should always be created, with default `admin` name and a generated password

Reply:
Should I remove the possibility for the user to set different credentials?

Reply:
What do you mean by "different credentials"?

The user can supply a password and/or provide OIDC configuration. Is there something else?

Reply:
got it. That was the question ;)